File: ui/qnote/src/app/tools/models/Cours.ts
export class Cours {
name: string;
code: string
note_total: number;
note_pass:  number;
note_rep:   number;
note_exc:   number;
is_calc: boolean;
is_number: boolean;
map = 300;
public constructor(_code, _name, _note_total, _note_pass, _note_rep,_note_exc,ic, is_number=true ) {
    this.code = _code;
    this.name = _name;
    this.note_total = _note_total;
    this.note_pass  = _note_pass;
    this.note_rep = _note_rep;
    this.note_exc = _note_exc;
    this.is_calc = ic;
    this.is_number = is_number;
    this.map = 300;
  }
}

File: api/BysApp/src/main/java/Palmares/Cours.java
package Palmares;

import lombok.Data;

@Data
public class Cours {  	
   Long id;
   String code;
   String name;
   private int coef ; 
   private float note_total;
   int nbre_res;

public Cours(Long id, String code, String name, int coef, float note_total) {
	super();
	this.id = id;
	this.code = code;
	this.name = name;
	this.coef = coef;
	this.note_total = note_total;
}

public Cours(Long id, String code, String name, int coef, float note_total, int nbre_res) {
	super();
	this.id = id;
	this.code = code;
	this.name = name;
	this.coef = coef;
	this.note_total = note_total;
	this.nbre_res = nbre_res;
}







   
}
